### PR TITLE
Added config option for database port

### PR DESCRIPTION
**Pull request by geordish**
Fix for #5514

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: poller-wrapper.py [hunk 1]
```diff
@@ -86,7 +87,6 @@ def get_config_data():
     db_port = int(config['db_host'].rsplit(':')[1])
 else:
```
Comment (geordish) [inline poller-wrapper.py]: I'm tempted to just remove this completely, as I don't see how it could work with the rest of librenms?

**Comment by laf** [comment]
`./LibreNMS/IRCBot.php` needs updating

Would also be good to do `./html/plugins/Weathermap/data-pick.php` from the plugins repo. Happy to do that.

Looks ok to me, needs testing quite a bit though.

**Comment by laf** [comment]
Also needs:

./poller-service.py
./discovery-wrapper.py

File: poller-wrapper.py (hunk 1 above)
Comment (laf) [inline poller-wrapper.py]: Probably a hangup from when we used mysql_connect which I think did/does accept :port

**Comment by laf** [comment]
Running it in test install now.

**Comment by laf** [comment]
@librenms/reviewers Works for me. Anyone else want to test?

**Comment by VimCommando** [comment]
If you run into an issue like I did, where `poller-wrapper.py` tells you it cannot connect to the database, but `validate.php` and `poller.php` say the DB is OK; double-check your `db_host` string. If you included the port previously (i.e. "localhost:3306") this update will break that.

The fix is to simply define `db_port` separately in config.php and `poller-wrapper.py` should work fine again.

**Comment by laf** [comment]
Sorry that this might have broken something for you - out of interest how did you have poller.php working on the non-standard port or is it just that you'd specified :3306 all along?

**Comment by VimCommando** [comment]
I just had the default port specified in my URL string, even though I didn't need to.

**Pull request merged**